examples/jsm/webxr/ARButton.js

```diff
@@ -97,40 +96,54 @@ var ARButton = {
 
 		function stylizeElement( element ) {
 
-			element.style.position = 'absolute';
-			element.style.bottom = '20px';
-			element.style.padding = '12px 6px';
-			element.style.border = '1px solid #fff';
-			element.style.borderRadius = '4px';
-			element.style.background = 'rgba(0,0,0,0.1)';
-			element.style.color = '#fff';
-			element.style.font = 'normal 13px sans-serif';
-			element.style.textAlign = 'center';
-			element.style.opacity = '0.5';
-			element.style.outline = 'none';
-			element.style.zIndex = '999';
+			const addedStyles = {
```

Comment:
Why did you change this?

Reply:
Eres rapido! :P
I think its more encapsulated than straight manipulation of the original object, define the to-be-added properties with a nice name that the reader knows the purpose in advance and then iterate thru it, ready for a (if) a bigger refactor.

Reply:
I don't think this change should be part of this PR either.

Reply:
got it, will PR any additional changes on a different one and maybe an extra feature